.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
```

Comment:
Is it for the generated Docker images? Otherwise, the project itself has no version requirements for its dependencies, so they should always be up-to-date.